### DISCUSSION TITLE
13 luglio 2023

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/mdp2223#8] [Q&A] [answered]
In questa discussion a tema la traccia di esame del 13 luglio 2023 discuteremo di eventuali proposte di soluzione all'esame in questione

**Comment by Gio0203c** [comment, marked as answer]
[13_07_2023 - COMPLETO.pdf](https://github.com/sapienzastudentsnetwork/mdp2223/files/12988622/13_07_2023.-.COMPLETO.pdf)

**Comment by FeddyLix17** [comment]
[Walkthrough Esame 13 Luglio 2023](https://github.com/FeddyLix17/Computer-Science/tree/main/Primo%20Anno/Secondo%20Semestre/Metodologie%20di%20Programmazione/Esami%20precedenti/2023/Luglio/Esame%2013%20Luglio%202023)

**Comment by Frisbon** [comment]
Allego la traccia d'esame senza soluzione (non capisco perché non le mettano lol)
[MDP_Esame_13_Luglio_2023.pdf](https://github.com/sapienzastudentsnetwork/mdp2223/files/13962372/MDP_Esame_13_Luglio_2023.pdf)
